llvmDebugInfoC/src/scripts/konan_lldb.py

```diff
@@ -23,34 +23,48 @@
 
 import lldb
 import struct
+import os
+
+def is_appcode():
+    return 'AppCode' in os.getenv('XPC_SERVICE_NAME', '')
```

Comment:
Hacky, but seems to work. 'XPC_SERVICE_NAME' has 'AppCode' in the string. Should probably be something more formal.